src/gleam/erlang/os.gleam

```diff
@@ -63,3 +63,48 @@ pub external fn set_env(name: String, value: String) -> Nil =
 ///
 pub external fn unset_env(name: String) -> Nil =
   "gleam_erlang_ffi" "unset_env"
+
+/// Run a command in a command shell.
+///
+/// This function returns the output of the command as a String.
+///
+/// ## Examples
+///
+///    > cmd("ls")
+///    "Code\nDesktop\nDocuments\nMusic\nPictures\nVideos"
+///
+///    > cmd("ls -a")
+///    ".\n..\n.asdf\n.config\nCode\nDesktop\nDocuments\nMusic\nPictures\nVideos"
+///
+pub external fn cmd(String) -> String =
```

Comment:
I think we would want to think carefully about the API of this function. It can fail so it ought to return a result, and it is easy to accidentally create zombie processes when used directly like this. https://hexdocs.pm/elixir/Port.html#module-zombie-operating-system-processes

We would also want to use a different name as the stdlib doesn't use any abbreviations for function names.

Reply:
Yes, you're right. I was a bit uneasy while writing it that the result doesn't convey any sense of whether the program appears to have succeed (inferring from exit code 0) or failed. I'll reconsider its API.

Reply:
Name updated. While Erlang's `os:cmd` doesn't open a port, Elixir's `System.cmd` does. It doesn't appear that it's possible to get the return code from `os:cmd` alone, so to provide rich return information I'll also need to open a Port here, with implied consequences.